src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -0,0 +1,116 @@
+package ui.components;
+
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import prefs.Preferences;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * a central place to specify keyboard shortcuts
+ *
+ * Classes that currently have keyboard shortcut code:
+ * ui.components.NavigableListView
+ * ui.issuecolumn.ColumnControl
+ * ui.issuepanel.IssuePanel
+ * ui.MenuControl
+ *
+ * Utility Class:
+ * util.KeyPress
+ */
+public class KeyboardShortcuts {
```

Comment:
I meant cases like both E and R can be used to mark an issue as read. I guess we should not allow users to use the same shortcut for multiple things. In such cases, we go by the order in which they are specified and take the one that comes later in the order. i.e. every time a shortcut is specified, we override the one specified previously.